benchmarks/Microsoft.AspNetCore.Routing.Performance/Matching/JumpTableMultipleEntryBenchmark.cs

```diff
@@ -67,15 +64,24 @@ public int Baseline()
                 var @string = strings[i];
                 var segment = segments[i];
 
-                destination = segment.Length == 0 ? -1 :
-                    segment.Length != @string.Length ? 1 :
-                    string.Compare(
+                if (segment.Length == 0)
+                {
+                    destination = -1;
+                }
+                else if (segment.Length != @string.Length)
+                {
+                    destination = 1;
+                }
+                else
+                {
+                    destination = string.Compare(
                         @string,
                         segment.Start,
                         @string,
                         0,
-                        @string.Length,
+                        segment.Length,
                         StringComparison.OrdinalIgnoreCase);
+                }
```

Comment:
This code should do the same thing, but it's now a bit clearer that it's correct in the context of a jump table.